WcaOnRails/spec/models/user_spec.rb

```diff
@@ -498,8 +500,8 @@
   end
 
   it "#teams and #current_teams return unique team names" do
-    wrc_team = Team.find_by_friendly_id('wrc')
-    wrt_team = Team.find_by_friendly_id('wrt')
+    wrc_team = Team.wrc
+    wrt_team = Team.wrt
```

Comment:
As above, I think it's better to replace all `wrc_team` directly with `Team.wrc` (same for wrt)